lib/Alchemy/Phrasea/TaskManager/Editor/SubdefsEditor.php

```diff
@@ -70,7 +74,115 @@ protected function getFormProperties()
             'flush' => static::FORM_TYPE_INTEGER,
             'maxrecs' => static::FORM_TYPE_INTEGER,
             'maxmegs' => static::FORM_TYPE_INTEGER,
-            'embedded' => static::FORM_TYPE_BOOLEAN
+            'embedded' => static::FORM_TYPE_BOOLEAN,
         ];
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function updateXMLWithRequest(Request $request)
```

Comment:
instead of completely override this method, what about making it usable for other editors by making changes to the parent method?